File: algorithm-programmers-greedy-lifeboat/src/test/java/com/currenjin/SolutionTest.java
package com.currenjin;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class SolutionTest {
    @Test
    void sample1() {
        assertEquals(3, new Solution().solution(new int[]{70, 50, 80, 50}, 100));
    }

    @Test
    void sample2() {
        assertEquals(3, new Solution().solution(new int[]{70, 80, 50}, 100));
    }
}

File: algorithm-programmers-greedy-lifeboat/src/test/kotlin/com/currenjin/SolutionTest.kt
package com.currenjin

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class SolutionTest {
    @Test
    fun sample1() {
        assertEquals(3, Solution().solution(intArrayOf(70, 50, 80, 50), 100))
    }

    @Test
    fun sample2() {
        assertEquals(3, Solution().solution(intArrayOf(70, 80, 50), 100))
    }
}
